components/About/styles.module.scss

```diff
@@ -41,5 +60,6 @@
 }
 
 .text {
-    margin-left: 100px;
+    margin: 0 auto;
```

Comment:
Now that you're using `display: grid` for the layout, spacing would better be handled with `grid-gap`